docs/howto/windows.md

```diff
@@ -0,0 +1,30 @@
+Small guide on how to use the terminal using WSL (Windows Subsystem for Linux) with the windows file system, and using it to perform any commands while running a react native android app in Android emulator on Windows.
+
+### Install WSL
+Install your preferred distribution of Linux available in the Windows store (You must have Windows build 1607 or later to be able to do this). I used Ubuntu for this guide.
+
+### Accessing the filesystem
+You can access your Windows local drives by navigating to `/mnt/<drive-letter>`
+
+### Running npm commands
+First install node in your subsystem. For example, to install node 8.x in Ubuntu, run:
+
+`curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -`
+`sudo apt-get install -y nodejs`
+
+Install `react-native-cli` globally with `npm i -g react-native-cli`.
+
+### Modify PATH variable
+Run the following commands so the subsystem checks for npm in `/usr/bin` before it checks `C\Program Files` and tries to run the Windows installation (which of course, does not work in Linux).
+
+`export PATH="$HOME/bin:$HOME/.local/bin:/usr/bin:$PATH"`
+`source ~/.profile`
+
+### Run emulator
```

Comment:
Now would be a good place to tell the developer to run `yarn`.
They need to install the dependencies before running the project.

They can also run `npm` that they have installed with `node` but we recommend using `yarn`.

That means a previous step would be to explain how to install `yarn`.